tasks/methodology/word-embeddings/MLproject

```diff
@@ -0,0 +1,12 @@
+name: word_embedding
+
+conda_env: conda.yaml
+
+entry_points:
+  main:
+    parameters:
+      word1: string
+      word2: string
+      topn: {type: integer, default: 10}
+
+    command: "python word_embedding.py {word1} {word2} {topn}"
```

Comment:
actual program name is embeddings.py there is extra `s` in file present in repository. Please update here.

Reply:
I have resolved the comment and checked in the updated code.